src/main/java/com/labsynch/labseer/service/BulkLoadServiceImpl.java

```diff
@@ -866,6 +866,9 @@ public Parent validateParent(Parent parent, String chemist, Collection<BulkLoadP
 			dupeParentList = chemStructureService.searchMolStructures(parent.getCmpdRegMolecule(), StructureType.PARENT, ChemStructureService.SearchType.DUPLICATE_TAUTOMER, -1F, -1);
 		}
 		if (dupeParentList.length > 0) {
+			// If we find a duplicate parent, we want to maintain the multiple fragments boolean we already have to save time since we have already checked for this
+			// and it is a transient property which wil be null on the found parent we may be overwriting with
+			boolean multipleFragments = parent.getMultipleFragments();
```

Comment:
I think technically this should be a `Boolean` (allowing for null values) even though we don't expect this to be null at this point.

Reply:
Thanks, should have noticed the underlying type. Fixed it.